src/components/maze/GameboardContainer.tsx

```diff
@@ -280,15 +282,16 @@ export function GameboardContainer({
   const shareReferralLink =
     'https://' +
     new URL(window.location.href).host +
-    `?referralId=${addresses['near']}`;
+    `?referralId=${addresses[blockchain]}`;
 
-  const notAllowedToPlay = addresses['base']
-    ? false
-    : (!isUserNadabotVerfied &&
-        !isUserHolonymVerified &&
-        earnedButNotMintedCheddar >= 100) ||
-      (!hasEnoughBalance && earnedButNotMintedCheddar >= 100) ||
-      (!hasEnoughBalance && totalMintedCheddarToDate >= 100);
+  const notAllowedToPlay =
+    blockchain === 'base'
```

Comment:
Why whould you ask if the current network is base instead of if the user have a logged account in base?

Reply:
we don't have any human verification checks for `base` blockchain as of now, so it wouldn't matter if user is logged in or not. and we use `notAllowedToPlay` when user is logged in.